compositionspace/datautils.py

```diff
@@ -222,29 +201,42 @@ def read_apt_to_df(self):
 
         pbar = tqdm(os.listdir(self.params["input_path"]), desc="Reading files")
         for filename in pbar:
+            
+            if filename.endswith(".POS"):
+                #print(filename)
+                path = os.path.join(self.params["input_path"], filename)
+                pos = self.get_pos(path)
+                df_POS_MASS = pd.DataFrame({'x':pos['x'],'y': pos['y'],'z': pos['z'],'Da': pos['m']})
+                df_Mass_POS_lst.append(df_POS_MASS)
+                file_name_lst.append(filename)
+            
             if filename.lower().endswith(".pos"):
                 path = os.path.join(self.params["input_path"], filename)            
-                pos = self.read_pos(path)
+                pos = self.get_pos(path)
                 df_POS_MASS = pd.DataFrame({'x':pos['x'],'y': pos['y'],'z': pos['z'],'Da': pos['m']})
                 df_Mass_POS_lst.append(df_POS_MASS)
                 file_name_lst.append(filename)
 
             if filename.endswith(".apt"):
                 path = os.path.join(self.params["input_path"], filename)
-                POS_MASS = self.read_apt(path) 
+                POS_MASS = self.get_apt(path) 
                 df_POS_MASS = pd.DataFrame(POS_MASS, columns = ["x","y","z","Da"])
                 df_Mass_POS_lst.append(df_POS_MASS)
                 file_name_lst.append(filename)
 
             if filename.lower().endswith(".rrng"):
                 path = os.path.join(self.params["input_path"], filename) 
-                ions,rrngs = self.read_rrng(path)
+                ions,rrngs = self.get_rrng(path)
+                
+            if filename.endswith(".RRNG"):
+                path = os.path.join(self.params["input_path"], filename) 
+                ions,rrngs = self.get_rrng(rrange_file)
                 
         return (df_Mass_POS_lst, file_name_lst, ions, rrngs) 
 
 
 
-    def chunkify_apt_df(self):
+    def get_big_slices(self):
```

Comment:
I am completely fine with function name changes, just please also update in the tests (in the tests folder), and it should be fine.

Reply:
I will change this

Reply:
This is also true for a number of other functions, you could either put back the names; or change in tests, whichever you prefer. From my side, both are ok.